Cython/Compiler/PyrexTypes.py

```diff
@@ -3780,31 +3780,39 @@ def can_coerce_from_pyobject(self, env):
             return True
         return False
 
+    def _get_conversion_tags_context_cls(self, env, tags, context):
+        # modifies tags and context, returns the cls name or False (if it's not convertible)
+        for ix, T in enumerate(self.templates or []):
+            if ix >= builtin_cpp_conversions[self.cname]:
+                if not isinstance(T, TemplatePlaceholderType):
+                    tags.append(T.specialization_name())
+                continue
+            if not T.create_to_py_utility_code(env):
+                return False
+            tags.append(T.specialization_name())
+
+        context['Tp'] = self
```

Comment:
The method is called `_get_…` but changes its input arguments. That seems fishy. There's more than one function in here, I'd say.